gui/src/main/java/org/jboss/as/console/client/shared/deployment/DeploymentStep2.java

```diff
@@ -64,7 +64,11 @@ public Widget asWidget() {
         TextBoxItem nameField = new TextBoxItem("name", Console.CONSTANTS.common_label_name());
         TextBoxItem runtimeNameField = new TextBoxItem("runtimeName", Console.CONSTANTS.common_label_runtimeName());
         CheckBoxItem enable = new CheckBoxItem("enableAfterDeployment", "Enable");
-        form.setFields(nameField, runtimeNameField, enable);
+        if (Console.getBootstrapContext().isStandalone()) {
+            form.setFields(nameField, runtimeNameField, enable);
+        } else {
+            form.setFields(nameField, runtimeNameField);
```

Comment:
Because in HAL version older than 2.4.3, it automatically enabled app after replacement. 
Now, if user uncheck "Enable", it will only remove old content and add content inside server-xxx/data/content . The old deployment is still there and accessible before enabling the new deployment. I think this is unwanted behaviour to expose this checkbox just in domain mode.

Also, I assume it used if-else on purpose to check server mode before I changed it in https://github.com/hal/core/commit/83e37fbd365ab7f307c1d5f09412809ecb796462 . I now pass "true" in NewDeploymentWizard.java for domain mode.